Code/GraphMol/TautomerQuery/TautomerQuery.h

```diff
@@ -35,7 +36,7 @@ class RDKIT_TAUTOMERQUERY_EXPORT TautomerQuery {
 
   // tests if a match to the template matches a specific tautomer
   bool matchTautomer(const ROMol &mol, const ROMol &tautomer,
-                     const MatchVectType &match,
+                     const std::vector<unsigned int> &match,

```

Comment:
why did you switch from `MatchVectType` here? I know it's not ideal, but it's at least somewhat consistent.

Reply:
I'm using an extraFinalCheck method in the parameters for template substructure search to see if any of the tatutomers actually match.  The signature for that uses a vector rather than MatchVectType, so I changed it in matchTatutomer (the method is private and this is the only usage)

Reply:
makes sense